councilmatic_core/models.py

```diff
@@ -1,96 +1,110 @@
-from datetime import datetime
-import inspect
-import importlib
-
-import pytz
+import datetime
+import os
 
 from django.db import models
 from django.core.exceptions import ImproperlyConfigured
 from django.conf import settings
-from django.contrib.postgres.fields.jsonb import JSONField
-from django.core.urlresolvers import reverse
-from django.core.urlresolvers import NoReverseMatch
+from django.urls import reverse, NoReverseMatch
 from django.utils import timezone
+from django.db.models import Case, When, Value
+from django.db.models.functions import Cast
+from django.utils.functional import cached_property
+from django.core.files.storage import FileSystemStorage
 
-if not (hasattr(settings, 'OCD_CITY_COUNCIL_ID') or hasattr(settings, 'OCD_CITY_COUNCIL_NAME')):
-    raise ImproperlyConfigured(
-        'You must define a OCD_CITY_COUNCIL_ID or OCD_CITY_COUNCIL_NAME in settings.py')
+from proxy_overrides.related import ProxyForeignKey
+import opencivicdata.legislative.models
+import opencivicdata.core.models
 
-if not hasattr(settings, 'CITY_COUNCIL_NAME'):
-    raise ImproperlyConfigured(
-        'You must define a CITY_COUNCIL_NAME in settings.py')
+static_storage = FileSystemStorage(location=os.path.join(settings.BASE_DIR, settings.APP_NAME, 'static'), base_url='/')
```

Comment:
i could only get the downloaded headshots to work if they were saved to the app static directory, then moved to the static root via collectstatic.

Reply:
added a call to collectstatic to the end of this command